dockerfiles/tasks.py

```diff
@@ -1,11 +1,14 @@
+import os
+import sys
+
 from invoke import task
 
 DOCKER_COMPOSE = 'common/dockerfiles/docker-compose.yml'
 DOCKER_COMPOSE_SEARCH = 'common/dockerfiles/docker-compose-search.yml'
 DOCKER_COMPOSE_WEBPACK = 'common/dockerfiles/docker-compose-webpack.yml'
 DOCKER_COMPOSE_ASSETS = 'dockerfiles/docker-compose-assets.yml'
 DOCKER_COMPOSE_OVERRIDE = 'docker-compose.override.yml'
-DOCKER_COMPOSE_COMMAND = f'docker-compose -f {DOCKER_COMPOSE} -f {DOCKER_COMPOSE_OVERRIDE} -f {DOCKER_COMPOSE_SEARCH} -f {DOCKER_COMPOSE_WEBPACK}'
+DOCKER_COMPOSE_COMMAND = f'docker-compose -f {DOCKER_COMPOSE_OVERRIDE} -f {DOCKER_COMPOSE} -f {DOCKER_COMPOSE_SEARCH} -f {DOCKER_COMPOSE_WEBPACK}'
```

Comment:
this change isn't related to this PR

Reply:
I've done it because I need it to when calling the deploy script. Otherwise, the compose project is not read properly from .env file.

Reply:
As mentioned in https://github.com/readthedocs/readthedocs.org/pull/9319#discussion_r895914856, this change will break other things

Reply:
Cool! We can undo it before merging this PR, but we need to fix the `.env` issue for this PR to work.

Reply:
Note to myself: Revert this change once https://github.com/readthedocs/common/pull/148 got merged.